src/Uno.UI/UI/Xaml/INeedsThemeBindingUpdates.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Uno;
+
+namespace Windows.UI.Xaml
+{
+	/// <summary>
+	/// Implemented by types that have <see cref="DependencyObject"/> children that are not reachable via a <see cref="DependencyProperty"/>, which may
+	/// have theme resource bindings needing update when the app theme changes.
+	/// </summary>
+	internal interface INeedsThemeBindingUpdates
```

Comment:
Wondering if the need to access to hidden nested `DependencyObject` is exclusive to "ThemeBinding". I would go with a more generic name like `ICustomChildrenProvider` or `IAdditionalChildrenProvider`

Reply:
Makes sense, I've updated it.